scripts/generate-svm-assets.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Set paths for source and destination
+TARGET_IDL="./target/idl"
+TARGET_TYPES="./target/types"
+SVM_ASSETS="./src/svm/assets"
+SVM_IDL="$SVM_ASSETS/idl"
+SVM_TYPES="$SVM_ASSETS"
+IDL_OUTPUT_FILE="$SVM_IDL/index.ts"
+TYPES_OUTPUT_FILE="$SVM_TYPES/index.ts"
+
+# Ensure the destination directories exist
+mkdir -p "$SVM_IDL"
+mkdir -p "$SVM_TYPES"
+
+# --- Copy Files ---
+echo "Copying IDL files..."
+cp -r "$TARGET_IDL/"* "$SVM_IDL/"
+
+echo "Copying Types files..."
+cp -r "$TARGET_TYPES/"* "$SVM_TYPES/"
+
+# --- Generate IDL index.ts ---
+echo "Generating IDL index.ts..."
+> "$IDL_OUTPUT_FILE"
+
+# Add autogenerated file note
+{
+  echo "// This file has been autogenerated. Do not edit manually."
+  echo "// Generated by a script."
+  echo
+} >> "$IDL_OUTPUT_FILE"
+
+IMPORTS=""
+EXPORTS=""
+
+for file in "$SVM_IDL"/*.json; do
+  filename=$(basename -- "$file")
+  name="${filename%.json}"
+  camelCaseName=$(echo "$name" | awk -F'_' '{
+    for (i=1; i<=NF; i++) {
+      printf toupper(substr($i,1,1)) tolower(substr($i,2));
+    }
+  }')
+  IMPORTS="${IMPORTS}const ${camelCaseName}Idl = require(\"./${filename}\");\n"
```

Comment:
We use `require()` for the IDLs similarly as before in the scripts to avoid ts issues: https://github.com/across-protocol/contracts/blob/e0cc5a2801d53a72da5bb5db469da8cfdf3468ad/scripts/svm/executeRebalanceToHubPool.ts#L77